lumicks/pylake/kymotracker/detail/trace_line_2d.py

```diff
@@ -180,19 +182,25 @@ def _traverse_line_direction(
     return nodes
 
 
+@dataclass()
 class KymoLineData:
-    __slots__ = ["time_idx", "coordinate_idx"]
+    """Data class which holds the raw data for a Kymograph line.
```

Comment:
I feel like this comment is just restating the name + `@dataclass`. You might want to change it to explain why it is needed and what's its relationship with `KymoLine`.

Reply:
True. I've added a bit more description to the docstring.